### PR TITLE
Upgrade sdf to preview8 and run dataflow-composed examples

### PR DESCRIPTION
**Pull request by crajcan**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -11,11 +11,11 @@ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 rustup update
 ```
 
-#### Install Fluvio & SSDK
+#### Install Fluvio & SDF
 
 ```bash
 curl -fsS https://hub.infinyon.cloud/install/install.sh | bash
-fvm install ssdk-preview7
+fvm install sdf-preview7
```
Comment (ajhunyady) [inline README.MD]: preview-8

**Review by ajhunyady — COMMENTED** [review]
Looks good. Just one comment.

**Review by ajhunyady — APPROVED** [review]

**Comment by ajhunyady** [comment]
@crajcan, should we sync to `preview-9` then merge?

**Comment by morenol** [comment]
I think that we can merge this one and then update the other PR, I can do that by rebasing/squashing the other PR

**Comment by ajhunyady** [comment]
ok, merged.

**Pull request merged**